src/NuGet.Core/NuGet.CommandLine.XPlat/Strings.resx

```diff
@@ -202,7 +202,7 @@
     <value>Specifies a NuGet package source to use during the restore.</value>
   </data>
   <data name="Source_Description" xml:space="preserve">
-    <value>Specifies the server URL</value>
+    <value>Specifies the package source.</value>
```

Comment:
```suggestion
    <value>Package source (URL, UNC/folder path or package source name) to be pushed to. Defaults to defaultPushSource defined at NuGet.Config</value>
```

Reply:
`dotnet nuget` reuses the strings for multiple commands. E.g. this one is used for both push and delete. So I have adjusted the string to “use” instead of “push”.

Reply:
Thanks for the comments. We can always use more string resources if necessary.